docs/javascript-configuration.md

```diff
@@ -3,86 +3,95 @@ id: javascript-configuration
 title: Scope Javascript Agent configuration
 sidebar_label: Configuration
 ---
-> If these properties are manually configured, use `true` or `false` only for the boolean value.
+
+> If these properties are manually configured, use only `true` or `false` for boolean values.
 
 The behaviour of the Scope Agent can be modified using environment variables or a configuration file `scope.yml` in the source root of the project.
 
 ## Disabling Scope instrumentation
 
 Scope auto instruments your code and tests by default. You can disable this behaviour.
 
-### Using Environment Variables
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Environment Variable-->
 
 ```sh
 SCOPE_AUTO_INSTRUMENT=false
 ```
 
-### Using YAML Configuration file
+<!--YAML Configuration File-->
 
 ```yaml
 scope:
   instrumentation:
     enabled: false
```

Comment:
We have to change this to remove the `scope` indentation right?

Reply:
yes, I'll do that in another PR, as it is a big change